test/hack/resource-management/pkg/metrics/metrics.go

```diff
@@ -25,13 +25,12 @@ import (
 )
 
 const (
-	karpenterMetricRegion    = "us-east-2"
-	karpenterMetricDatabase  = "karpenterTesting"
-	karpenterMetricTableName = "sweeperCleanedResources"
+	karpenterMetricRegion   = "us-east-2"
+	karpenterMetricDatabase = "karpenterTesting"
 )
 
 type Client interface {
-	FireMetric(context.Context, string, float64, string) error
+	FireMetric(context.Context, string, string, float64, string) error
```

Comment:
Can we name these parameters now? It's getting a bit difficult to reason about what parameter is what